src/main/java/com/ir/example/miniticket/model/Gender.java

```diff
@@ -0,0 +1,42 @@
+package com.ir.example.miniticket.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum Gender {
+
+    MALE(0),
+    FEMALE(1),
+    UNDEFINED(2);
+
+    private Integer id;
+
+    Gender(Integer id) {
+        this.id = id;
+    }
+
+    @JsonValue
+    public int getId() {
+        return id;
+    }
+
+    /**
+     * Gets a Gender given it's id.
+     *
+     * @param id the Gender id.
+     * @return the Gender.
+     */
+    @JsonCreator
+    public static Gender valueOf(Integer id) {
+        if (id == null) {
+            return null;
+        }
+        int rawId = id;
```

Comment:
Why did you created the `rawId` variable? I thought it was because of Java autoboxing, but you use it in a comparison with `gender.id` that it is an `Integer`.

Reply:
Sorry! I just copy the same Enum of Bileto-User. I will fix in my project!